### ISSUE TITLE
Project will not compile with Zios Themes imported

### ISSUE THREAD

**Issue by gampixi** [zios/unity-themes#8]
```
236: [Unity] Unhandled Exception: System.Reflection.ReflectionTypeLoadException: The classes in the module cannot be loaded.
237: [Unity] Error building Player because scripts had compiler errors
238: [Unity] Player export failed. Reason: 2 errors
239: ! build of 'windows-desktop' failed. compile failed
```

The only errors that show up.

**Comment by gampixi** [comment]
The project compiles after removing Zios Themes, but now (probably because I forgot to set the style back to default) my toggle boxes are glitched in the editor.

http://i.imgur.com/ZGhqa3o.gifv

**Comment by bradsmithee** [comment]
Restarting Unity should fix your toggle box issue.  Strange that you have compile errors out of the box.  Are you using a checkout from the GIT or a release dll?

**Comment by gampixi** [comment]
I was using the release dll.

**Comment by bradsmithee** [comment]
Ah.  Forgot to mark the dll as editor only in the release.  I'll make sure it's fixed in the follow up patch release.  In the meantime, you can do : [img]http://bradsmithee.com/local/[2205][2016-08-09][0949].png[/img]

**Comment by gampixi** [comment]
Thanks!

2016-08-09 17:50 GMT+03:00 bradsmithee <notifications@github.com>:

**Comment by bradsmithee** [comment]
Resolved in [Unity 5.5 Hotfix](https://github.com/zios/unity-themes/releases/tag/r522).
